<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPrzepisDialog</class>
 <widget class="QDialog" name="AddPrzepisDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Składniki</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="skladnikiComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dodajSkladnikPushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Dodaj składnik</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="skladnikiListView">
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>651</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="stworzPrzepisPushButton">
      <property name="text">
       <string>Stwórz przepis!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nazwa</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nazwaLineEdit">
      <property name="maxLength">
       <number>200</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Czas przygotowania (minuty)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="czasPrzygotowaniaSpinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Trudność</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="trudnoscComboBox">
      <item>
       <property name="text">
        <string>ŁATWE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ŚREDNIE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TRUDNE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MISTRZ KUCHNI</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ulubione</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="ulubioneCheckBox">
      <property name="text">
       <string>Czy ulubione?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Opisz przygotowanie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="przygotowanieTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
